spec/request_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe UsersController, type: :controller do
```

Comment:
- [ ] Second, please create Post requests tests as well and it should cover `index` and `show` methods of controller.